### PR TITLE
Change `MATLAB Octave` to `MATLAB` to make the languages list look better

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Change `MATLAB Octave` to `MATLAB` to make the languages list look better.
  - MATLAB Octave can also be interpreted as `MATLAB`.

Updated languages list

### CODE REVIEW

File: lib/repositories.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const Repositories = {
     baseDir: "src",
   },
   aarch64_assembly: {
-    name: "AArch64 Assembly",
+    name: "AArch64",
```
Comment (ayaankhan98) [inline lib/repositories.ts]: can we write `x64 ASM`?

File: lib/repositories.ts [hunk 2]
```diff
@@ -108,7 +108,7 @@ export const Repositories = {
     baseDir: "src",
   },
   "matlab-octave": {
-    name: "MATLAB Octave",
+    name: "MATLAB",
```
Comment (ayaankhan98) [inline lib/repositories.ts:111]: LGTM!

File: lib/repositories.ts (hunk 1 above)
Comment (raklaptudirm) [reply lib/repositories.ts]: @Panquesito7 ?

**Comment by maltejur** [comment]
Reordering the repositories directly also leads to a different priority (if there now is a Ruby implementation for a algorithm, it will also be the first to be shown). This also leads to the build error, because on the main page we search for the algorithm `quick-sort`, but it is now called `quicksort`, because that is the name that gets resolved from the ruby filename.

If you want to change the order, you would probably have to do it here:
https://github.com/TheAlgorithms/website/blob/59ea172afaf12ff734b816fcb5439e33cf1df50a/pages/index.tsx#L131

But maybe we should rework the whole languages list completely.

**Comment by raklaptudirm** [comment]
@maltejur Changing the order in `index.tsx` would render `repositories.ts` useless. We do need to rework the languages list. Should I switch back to the working commit or will we do the rework in this `PR`? Also, what's your view on renaming `AArch 64 Assembly` to `x64 Asm`?

**Comment by maltejur** [comment]
> @maltejur Changing the order in `index.tsx` would render `repositories.ts` useless.

The file is also used in many other places, thats why this change broke some other things, but that doesn't really matter.

> Should I switch back to the working commit or will we do the rework in this `PR`? 

I have already worked on a new list yesterday and pushed the branch now here:
https://github.com/TheAlgorithms/website/tree/improve_languages_list
I will open a new PR for that, so we can close this one. Also, what do you think about the new list? (you can take a look [here](https://website-4jx1jwkzi-thealgorithms.vercel.app/)) I am not sure if it is the right approach, so I would love to hear what you think about it.

> Also, what's your view on renaming `AArch 64 Assembly` to `x64 Asm`?

There is a difference between x64 Assembly and AArch64. With the new approach, it also shouldn't matter that the text is longer.

**Pull request closed without merging**